src/util/appConfigHelper.js

```diff
@@ -162,6 +163,25 @@ export const setHidePayPal = () => {
   return true;
 };
 
+export const setLanguage = async language => {
+  const BASE_URL = window.location.origin;
+
+  if (!i18n.hasResourceBundle(language, 'translation')) {
+    const data = await fetch(
+      `${BASE_URL}/cleeng-translations/${language}/translations.json`
+    )
+      .then(response => {
+        return response.json();
+      })
+      .catch(() => {});
+    i18n.addResourceBundle(language, 'translation', data, true, true);
```

Comment:
If we changed the namespace to `'translations'` is that okey to have here `'translation'` as a namespace in the `hasResourceBundle` and `addResourceBundle`?